cob_bms_driver/src/fake_bms.py

```diff
@@ -10,12 +10,13 @@ class FakeBMS(object):
     def __init__(self):
         self.srv_charging = rospy.Service('~set_charging', SetBool, self.charging_cb)
         self.srv_relative_remaining_capacity = rospy.Service('~set_relative_remaining_capacity', SetFloat, self.relative_remaining_capacity_cb)
-        self.pub_voltage            = rospy.Publisher('/bms/voltage', Float64, queue_size = 1)
-        self.pub_current            = rospy.Publisher('/bms/current', Float64, queue_size = 1)
-        self.pub_remaining_capacity = rospy.Publisher('/bms/remaining_capacity', Float64, queue_size = 1)
-        self.pub_charge_capacity    = rospy.Publisher('/bms/full_charge_capacity', Float64, queue_size = 1)
-        self.pub_temparature        = rospy.Publisher('/bms/temperature', Float64, queue_size = 1)
-        self.pub_charging_state     = rospy.Publisher('/bms/battery_charging', Bool, queue_size = 1)        
+        self.poll_frequency          = rospy.get_param('poll_frequency_hz')
```

Comment:
Use the `default_param = rospy.get_param('default_param', 'default_value')` notation:

`self.poll_frequency         = rospy.get_param('poll_frequency_hz', 20.0)`